charts/xwiki/templates/solr-statefulset.yaml

```diff
@@ -0,0 +1,102 @@
+{{- $fullName := include "solr.fullname" . -}}
```

Comment:
We need to render this only if solr.enabled is true. Not deploy workload and service if external solr is disabled.

Reply:
Good catch, I will update this part :+1: